spec/stream_action_spec.rb

```diff
@@ -1,119 +0,0 @@
-require 'spec_helper'
-
-describe "Pubsubstub::StreamAction without EventMachine" do
```

Comment:
Yeah I'm not happy with not having this covered either. The problem is that I need a way to test HTTP streaming, because the server never close the connection, so the test would hang forever.